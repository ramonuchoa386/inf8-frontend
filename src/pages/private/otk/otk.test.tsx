import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { rendererCreate } from '../../../utils/test/test-utils';

import Page from '.';

describe('Otk', () => {
  test('Snapshot', () => {
    const page = rendererCreate(
      <MemoryRouter>
        <Page />
      </MemoryRouter>
    ).toJSON();
    expect(page).toMatchSnapshot();
  });
});
