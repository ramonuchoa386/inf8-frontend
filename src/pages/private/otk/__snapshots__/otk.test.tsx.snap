// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Otk Snapshot 1`] = `
.c4 {
  border: 1px solid #FF805E;
  color: #FF805E;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  background-color: transparent;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: bold;
  padding: 3px 9px;
  border-radius: 100px;
  display: inline-block;
  font-size: 14px;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4:hover {
  -webkit-transition: all 0.7s ease;
  transition: all 0.7s ease;
  color: #FFF;
  background-color: #FF805E;
}

.c4:not(:last-of-type) {
  margin-right: 12px;
}

.c2 {
  width: 100%;
  background-color: #CFCFC4;
  padding: 17px 20px;
  -webkit-transition: all 0.4s ease-in-out;
  transition: all 0.4s ease-in-out;
  height: 54px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  overflow-y: auto;
}

.c5 {
  width: 100%;
  padding: 50px 20px 65px;
  height: 100vh;
}

.c5 code {
  color: #FF805E;
}

.c5 h3 {
  margin-bottom: 10px;
}

.c5 h5,
.c5 h6 {
  margin: 20px 0px 10px;
}

.c1 {
  width: 100%;
}

@media only screen and (min-width:768px) {
  .c2 {
    padding: 17px 60px;
  }
}

@media only screen and (min-width:1366px) {
  .c2 {
    padding: 17px 75px;
  }
}

@media only screen and (min-width:1024px) {
  .c0 {
    height: 100vh;
  }
}

@media only screen and (min-width:768px) {
  .c5 {
    padding: 50px 60px 65px;
  }
}

@media only screen and (min-width:1366px) {
  .c5 {
    padding: 50px 75px 65px;
  }
}

@media only screen and (min-width:768px) {
  .c1 {
    overflow-y: auto;
  }
}

<div
  className="c0"
>
  <div />
  <div
    className="c1"
  >
    <div
      className="c2"
      data-testid="HeaderID"
    >
      <div
        className="c3"
      >
        <span>
          Est√° Logado? 
          false
        </span>
        <div
          className=""
        >
          <button
            className="c4"
            onClick={[Function]}
          >
            Entrar
          </button>
        </div>
      </div>
    </div>
    <div
      className="c5"
    >
      <div>
        Otk
      </div>
    </div>
  </div>
</div>
`;
